libs/estdlib/src/gen_tcp.erl

```diff
@@ -44,7 +44,7 @@
     connect/3, send/2, recv/2, recv/3, close/1, listen/2, accept/1, accept/2, controlling_process/2
 ]).
 
--define(DEFAULT_PARAMS, [{active, true}, {buffer, 512}, {binary, true}, {timeout, infinity}]).
+-define(DEFAULT_PARAMS, [{active, true}, {buffer, 512}, {timeout, infinity}]).
```

Comment:
Indeed. I needed this for the test to pass with Erlang/OTP. Otherwise we would need to pass `{binary, false}` which doesn't work with Erlang/OTP (correct syntaxes are `list` which we support now, and `{mode, list}` which we don't).